js/playground/textTest.js

```diff
@@ -18,7 +18,8 @@ const button = $.makeButton(
     50,
 	"Multiple lined button!"
 )
-
+const font = $.loadCustomFont("New Font", "fonts/MightySouly-lxggD.ttf");
```

Comment:
Please change out this font for one we have a license to use maybe, https://dtinth.github.io/comic-mono-font/ since it is MIT.